static/src/javascripts/projects/common/modules/commercial/support-utilities.spec.js

```diff
@@ -1,9 +1,17 @@
 import { addCountryGroupToSupportLink } from './support-utilities';
-import { setGeolocation } from '../../../../lib/geolocation';
+import { getCookie } from '../../../../lib/cookies';
+
+jest.mock('../../../../lib/cookies', () => ({
+    getCookie: jest.fn(() => null),
+}));
 
 describe('addCountryGroupToSupportLink', () => {
+    beforeEach(() => {
+        getCookie.mockImplementation(() => null);
+    });
+
     test('adds country group to subscribe link', () => {
-        setGeolocation('GB');
+        getCookie.mockImplementation(() => 'GB');
```

Comment:
Does this need to change, if the cookie returns `null`? This should fall back to the storage value, doesn’t it?

Reply:
I don't think we need having a test case for localStorage anymore as cookie becomes the primary way. 
Testing the fallback logic is more related to geolocation logic itself rather than `support-utilities.spec.js`